index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Apple</title>
+	<link rel="stylesheet" href="./stylesheets/reset.css">
+	<link rel="stylesheet" href="./stylesheets/main.css">
+  </head>
+
+  <body>
+
+  	<div class="main-container flex-v">
+
+  		<div class="header">
+  			<div class="nav-bar">
+	  			<ul>
+	  				<li><a href="#">
+	  						<img class="logo-img" src="./images/apple-logo.png" alt="">
+	  					</a>
+	  				</li>
```

Comment:
- [ ] **Required** - These shouldn't be images as this is a project focused on backgrounds and gradients kindly ensure you make these backgrounds instead.

Reply:
Thanks @mikenath223. The images have been removed and used as backgrounds as instructed.